src/js/angular/core/directives/rdfresourcesearch/rdf-resource-search.directive.js

```diff
@@ -36,31 +36,26 @@ function rdfResourceSearchDirective($rootScope, $timeout,
       $scope.showInput = function () {
         element.find('.search-rdf-btn')
             .addClass('hidden-xs-up');
-        element.find('.search-rdf-input').css('top', '20px');
+        element.find('.search-rdf-input').addClass('show-rdf-search-box');
         $timeout(function () {
-          element.find('.close-rdf-search-btn')
-              .removeClass('hidden-xs-up');
-          element.find('search-resource-input .view-res-input')
-              .focus();
+          element.find('.close-rdf-search-btn').removeClass('hidden-xs-up');
+          element.find('search-resource-input .view-res-input').focus();
           window.addEventListener('mousedown', onWindowClick);
         }, 200);
       };
 
       $scope.hideInput = function () {
-        element.find('.search-rdf-input').css('top', '-550px');
-        element.find('.close-rdf-search-btn')
-            .addClass('hidden-xs-up');
+        element.find('.search-rdf-input').removeClass('show-rdf-search-box');
+        element.find('.close-rdf-search-btn').addClass('hidden-xs-up');
         $timeout(function () {
-          element.find('.search-rdf-btn')
-              .removeClass('hidden-xs-up')
-              .css('z-index', '3');
+          element.find('.search-rdf-btn').removeClass('hidden-xs-up');
           element.blur();
           window.removeEventListener('mousedown', onWindowClick);
         }, 200);
       };
 
       function onWindowClick(event) {
-        if (!(event.target.id === "search-box" || $(event.target).parents("#search-box").length)) {
+        if (!(event.target === "search-box" || $(event.target).closest("#search-box").length)) {
```

Comment:
Did you remove the `id` here by mistake?

Reply:
Yes. I was trying some things.